etc/fle2_aead_generate_tests.py

```diff
@@ -17,13 +17,14 @@
     .bytes_written_expected = {bytes_written_expected}
 }},"""
 
+print("/* This file is generated by ./etc/fle2_aead_generate_tests.py. */")
```

Comment:
Resulting file needs to be updated with new generator script output.

Reply:
Done.